src/clients/inst_scc.rb

```diff
@@ -164,9 +164,11 @@ def get_available_addons
     end
 
     # register all selected addons
+    # back returns directly to the extensions selection
     def register_addons
       return false if init_registration == :cancel
-      registration_ui.register_addons(@selected_addons, @known_reg_codes)
+      ret = registration_ui.register_addons(@selected_addons, @known_reg_codes)
+      (ret == :back) ? :extensions : ret
```

Comment:
NP: I'd rather use

```ruby
ret = :extensions if ret == :back
ret
```

here as the `()`, `?` and `:` symbols make the line less readable...